src/epd4in2/constants.rs

```diff
@@ -1,39 +1,39 @@
 #[rustfmt::skip]
 pub(crate) const LUT_VCOM0: [u8; 44] = [
-0x00, 0x17, 0x00, 0x00, 0x00, 0x02,        
-0x00, 0x17, 0x17, 0x00, 0x00, 0x02,        
-0x00, 0x0A, 0x01, 0x00, 0x00, 0x01,        
-0x00, 0x0E, 0x0E, 0x00, 0x00, 0x02,        
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00,        
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00,        
+0x00, 0x08, 0x00, 0x00, 0x00, 0x02,
+0x60, 0x28, 0x28, 0x00, 0x00, 0x01,
+0x00, 0x14, 0x00, 0x00, 0x00, 0x01,
+0x00, 0x12, 0x12, 0x00, 0x00, 0x01,
+0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

Comment:
These are the offical LUTs from Waveshare (see the full review text for more infos)

Reply:
You can still add the new ones but let the old one there as well.
You have tested the new ones on the 4gray display or also on others?

Reply:
I don't have a 4gray display to test on. This is the first I've heard of them.

Reply:
with 4gray I mean the one that has black, white, grey1 and grey2? I thought thats also the one @mgottschlag has.
I though that is basically the new variant of the epd4in2.

Reply:
> As indicated by a comment below Ben Krasnow's blog post on quick refresh on this display, there are two versions of the display. I copied the LUTs from the example code for the greyscale Good Display display (does that mean that my B/W display properly supports greyscale? Yay!) and those result in proper black/white pixels, whereas the LUTs in this library currently only result in grey pixels.

Reply:
I'm still not sure about the different versions. Going to rewatch that vid. I haven't seen a distinction or any advertisement of "4 gray shades", like you do with the 3 color ones.

I'm happy to punt the LUT and version business to a new PR, and leave this one about quick refreshes. Or if we solve this, do it here.

Reply:
Both LUTs seem to work on the display I have.

Reply:
Sorry for the delay.

Two things are currently missing to finish this:
- Can you maybe add the new LUT behind a new feature gate similar to the type_a_alternative_faster_lut feature?
- Make a seperate trait for the quick refresh features and add a function there which does both

Reply:
Sep trait added